packages/react/src/components/TreeView/TreeNode.tsx

```diff
@@ -77,6 +84,9 @@ export type TreeNodeProps = {
   label: React.ReactNode;
   /**
    * Callback function for when the node receives or loses focus
+   *
+   *     @deprecated The `onNodeFocusEvent` prop for `TreeNode` has
+   *     been deprecated after the introduction of context. It will be removed in the next major release.
```

Comment:
```suggestion
   * @deprecated The `onNodeFocusEvent` prop for `TreeNode` has
   * been deprecated after the introduction of context. It will be removed in the next major release.
```